crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -300,10 +283,96 @@ impl Persistence {
             id: auction_id,
             orders,
             prices,
-            deadline,
             surplus_capturing_jit_order_owners,
         })
     }
+
+    /// Get competition winner.
+    pub async fn get_competition_winner(
```

Comment:
```suggestion
    pub async fn get_winning_solution(
```

We are not getting a winner, we are getting a solution.

Reply:
resolved